warehouse/packaging/utils.py

```diff
@@ -84,12 +84,12 @@ def render_simple_detail(project, request, store=False):
         f"{project.normalized_name}/{content_hash}.{project.normalized_name}.html"
     )
 
+    length = None
```

Comment:
It seems like we can still return the length even if we're not storing the file, correct? Any reason not to return it here? Seems like this would be undefined behavior otherwise.